source/transactions-convenient-api/tests/commit-writeconcernerror.yml

```diff
@@ -208,7 +180,10 @@ tests:
             writeConcern: ~
           command_name: commitTransaction
           database_name: admin
-    outcome: *outcome
+    outcome:
```

Comment:
As with my comment above, this `outcome` assertion should never have passed.

The last two tests in this file for error codes UnknownReplWriteConcern and UnsatisfiableWriteConcern should _not_ retry `commitTransaction` after the first failure, and I would not expect the server to actually apply the data. Drivers should ensure that they are executing the `outcome` block and asserting that the collection is still empty after the operation(s) executes.